### ISSUE TITLE
Support tufte package arbitrary margin content

### ISSUE THREAD

**Issue by inventionate** [jimhester/lintr#796]
The [tufte](https://github.com/rstudio/tufte) package provides a special "marginfigure" chunk, which leads to false lintr errors (it is fine to use HTML in there):

````r
```{marginfigure}
"The tidyverse is an opinionated collection of R packages designed for data science. 
All packages share an underlying design philosophy, grammar, and data structures."

<span style="float: right">--- Hadley Wickham</span>
```
````

Produces this error message:

```
02-tidyverse.Rmd:26:1: error: Unexpected '<'
<span style="float: right">--- Hadley Wickham</span>
^
```

I hope this can be fixed. Anyway, thank you very much for your great package!

**Comment by MichaelChirico** [comment]
are you getting this error in the latest b development version?

**Comment by inventionate** [comment]
Unfortunately yes.
